RecoTauTag/HLTProducers/src/HLTPFDiJetCorrCheckerWithDiTau.cc

```diff
@@ -77,7 +79,7 @@ void HLTPFDiJetCorrCheckerWithDiTau::produce(edm::StreamID iSId, edm::Event& iEv
       const reco::PFJet& myPFJet1 = pfJets[iJet1];
       const reco::PFJet& myPFJet2 = pfJets[iJet2];
 
-      if ((myPFJet1.p4() + myPFJet2.p4()).M() < mjjMin_)
+      if (mjjMin_ <= 0. || (myPFJet1.p4() + myPFJet2.p4()).M2() < m2jjMin_)
```

Comment:
```suggestion
      if (mjjMin_ < 0. || (myPFJet1.p4() + myPFJet2.p4()).M2() < m2jjMin_)
```
(preserves current behaviour for case `mjjMin_ == 0`)